distribute/publish/articles/01-publishing-your-app/02-preparing-your-app.markdown

```diff
@@ -312,29 +313,38 @@ keep in mind:
       Security Manager. If you're enabling the security manager, you'll also
       need to define your Portal Access Control List (PACL) in this file.  Read
       [Plugins Security and PACL](/develop/tutorials/-/knowledge_base/6-2/plugin-security-and-pacl)
-      for information on developing secure apps. 
+      for information on developing secure apps.
 - *Deployment contexts*:
     - Liferay reserves the right to deny an application if any of its plugin
       deployment contexts is the same as a context of another plugin in the
-      Marketplace. 
+      Marketplace.
     - Liferay reserves the right to replace app plugin WAR files that have
       the same deployment context as plugins built by Liferay.
 
+<b>For Liferay 7 uploads</b>
+
+- **JAR (`.jar`) and WAB (`.wab`) files*:
```

Comment:
@enochchu Here don't you mean that for Liferay 7 you can upload module JARs (.jar) and WARs (.war files) for WAR-style plugins that have been adapted (not modularized) to Liferay 7?